doc/admin-guide/plugins/cache_range_requests.en.rst

```diff
@@ -144,6 +143,35 @@ option must have the same value (or not be defined) in order to work.
 
 Presence of the `--ims-header` automatically sets the `--consider-ims` option.
 
+X-Crr-Ident header support
+--------------------------
+
+.. option:: --consider-ident
+.. option:: -d
+.. option:: --ident-header=[header name] (default: X-Crr-Ident)
+.. option:: -j
+
+This supports the slice plugin which makes multiple adjancent
+range requests.  The slice plugin will record the identifier
+of the first range request (Etag, or Last-Modified in that order)
+and will add the value of that to the this header.
+
+.. code::
+
+    X-Crr-Ident: Etag: "foo"
+    X-Crr-Ident: Last-Modified: Tue, 19 Nov 2019 13:26:45 GMT
+
+If during the cache lookup hook a range request is considered stale
+the identifier from this header will be compared to the stale cache
+response and if matches the response will be marked FRESH to bypass
+visiting a parent cache with an IMS request.
```

Comment:
is it worth mentioning that this doesn't apply if the ims header is configured? we only check the identity header if we're not forced revalidating